src/components/charts/VoltageChart.svelte

```diff
@@ -12,7 +12,29 @@
   let voltageData = [];
   let chargingData = [];
 
-  $: if (voltageChart && voltageData && chargingData) {
+  $: getVoltageData(readings);
+
+  function getVoltageData(readings) {
+    voltageData = readings.reverse().map((reading) => {
+      const d = parseISO(reading.captured, DATE_TIME_FORMAT_KEY);
+      return {
+        x: format(d, DATE_TIME_FORMAT_KEY),
+        y: parseFloat(reading.voltage).toFixed(2),
+      };
+    });
+    chargingData = readings
+      .filter((reading) => reading.charging)
+      .map((reading) => {
+        const d = parseISO(reading.captured, DATE_TIME_FORMAT_KEY);
+        return {
+          x: format(d, DATE_TIME_FORMAT_KEY),
+          y: 6,
```

Comment:
that's sneaky :-). Quick effective way to get the full vertical bars

Reply:
Believe me, I looked for other potential solutions including adding the annotation plugin, but in the end this worked (and I think it's similar to what the Grafana charts are doing to indicate the same thing). Good thing we know voltage will never get much above 5V :)

Reply:
I like it! It's good practice to have constants like this declared as const variables, e.g. `const MAX_VOLTAGE = 6` to make the intent clear.

Reply:
Good suggestion. Added